### ISSUE TITLE
Errors in Argentina provinces (N2) in the WDBII_Borders_Level_02.

### ISSUE THREAD

**Issue by Esteban82** [GenericMappingTools/gshhg-gmt#4]
See https://github.com/GenericMappingTools/gmt/issues/4032 for details.

In order to fix the border a should replace WDBII_Borders_Level_02.txt (in WDBII/borders/) with the new file, right?

**Comment by PaulWessel** [comment]
Yes, you would have to determine which lines to replace since there is very little meta-data in those files.  You could use gmtspatial -D to help you find them since they will be near-duplicates at some threshold.

**Comment by Esteban82** [comment]
Ok. I am seeing it. I think I will have no problem to update the data.

What should I do with the header? How should I update it? N = number of vertices? I can't get the meaning of G, L, E, S, A.
`> Id=413 N = 553 G = 0 L = 2 E = 180 S = 0 R = 289.758056/292.638611/-32.392500/-31.954722 A = 0`

Also, there is a new segment (an small one) which surround the Federal city of Buenos Aires (similar to Washington DC). Should I add the segment?

**Comment by PaulWessel** [comment]
Yes, N is number of points, so change that.  G means "Greenwich" and I think for all your data it will be the same as it is now (0).  if you were doing the UK it may need to be 1.  L is just the level, so 2 as it is now.  E is 180 for all items except the Eurasia polygon which goes from -15 W to 192 E so we dont want to wrap at 180.  Not affecting your data though. S is for source.  0 is WDBII so you might as well set it to S = 1 as (other). R is the tight bounding box for that feature.  I use FORMAT_FLOAT_OUT=%.6 I think to get those 6 decimals the right way.  A is Area which is 0 for lines.
Yes, you can add another segment.  Make sure its Id is unique in that file.  Looks like the largest ID is 2011 so 2012 will work.

**Comment by Esteban82** [comment]
Ok. Thanks. I think I will have no problem with that.

Another question. I see that most of the segment fully extends between two provinces. But there are some segments that extends between several provinces (so maybe could be split) and there are others that do not full extends (so it can be joined to another). Is this necessary? I think not. It would the plot perfectly by GMT but I would like to confirm this.

**Comment by PaulWessel** [comment]
It just reflects the segmentation of the original data.  Yes, it would be better if it only were between two provinces but in the end they are just lines so make little difference I think,
